From philips@vnet.ibm.com Tue Aug  1 10:24:16 2006
Date: Tue, 1 Aug 2006 14:04:17 -0500
From: Brandon Philips <brandon@ifup.org>
To: gregkh@suse.de
Subject: genhd.c reference in Documentation/kobjects.txt
Message-ID: <20060801190417.GB23303@vnet.ibm.com>
Content-Disposition: inline

block/genhd.c no longer in drivers/.  Update Documentation/kobjects.txt

Signed-off-by: Brandon Philips <brandon@ifup.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 Documentation/kobject.txt |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/Documentation/kobject.txt
+++ gregkh-2.6/Documentation/kobject.txt
@@ -247,7 +247,7 @@ the object-specific fields, which includ
 - default_attrs: Default attributes to be exported via sysfs when the
   object is registered.Note that the last attribute has to be
   initialized to NULL ! You can find a complete implementation
-  in drivers/block/genhd.c
+  in block/genhd.c
 
 
 Instances of struct kobj_type are not registered; only referenced by
